src/main/java/fr/techad/edc/popover/internal/swing/components/Popover.java

```diff
@@ -100,6 +99,20 @@ public void windowLostFocus(WindowEvent e) {
         });
     }
 
+    public void hideOnHover(){
```

Comment:
Rename it. The goal of this method is to register a listener which closes the popover on lost focus and you name the method: hideOnHover.

A valid name should be: enableCloseOnLostFocus or registerCloseOnLostFocusListener.